src/api/EscargotPublic.h

```diff
@@ -744,6 +744,7 @@ class ESCARGOT_EXPORT DebuggerOperationsRef {
             return m_offset;
         }
 
+        StringRef* eval(StringRef* sourceCode, bool* isError);
```

Comment:
`isError` could be made mandatory with `bool&` if you prefer that

Reply:
Sorry for slow reponse. I prefer "&"
because, User can be confused by `isError` parameter must be filled with non-zero address or not